src/datascience-ui/react-common/seti/seti.less

```diff
@@ -0,0 +1,656 @@
+@font-face {
```

Comment:
Did you copy this from somewhere? If so we need to add a license for it to the ThirdPartyRespository.txt

Reply:
It's the seti icons that VS Code ships. Needed this for the language icon in the breadcrumbs at the top of the data viewer. This is the one that VS Code has: https://github.com/microsoft/vscode/blob/main/extensions/theme-seti/ThirdPartyNotices.txt

Reply:
Do we add the license manually, or is there a component governance autogeneration step somewhere? And do we need to add it to both ThirdPartyNotices-Distribution as well as ThirdPartyNotices-Repository?

Reply:
Updated Repository notice and manually added this dependency to AzDO CG